packages/db/src/index.ts

```diff
@@ -61,6 +61,7 @@ export {
   translateApiFiletoPostgresFile,
 } from './translate/file';
 export { translateApiCollectionToPostgresCollection } from './translate/collections';
+export { translatePostgresPdrToApiPdr } from './translate/pdr';
```

Comment:
We already have a line below exporting from `./translate/pdr`. We should add this export there

Reply:
https://github.com/nasa/cumulus/pull/2273/commits/7ed58e6a1895c91c2d21825e119ebfa9ace452bc